templates/fragmenter-config-pod.json

```diff
@@ -21,6 +21,11 @@
 	]
   },
   "transformers": [
+    {
+      "@type": "QuadTransformerBlankToNamed",
+      "searchRegex": "^(b[0-9]*_tagclass)",
+      "replacementString": "http://localhost:3000/www.example.org/$1"
```

Comment:
`www.example.org` seems a bit arbitrary :p
Perhaps we can rewrite to `http://localhost:3000/tags` or something like that?

Reply:
yes! I agree, I should have asked here :sweat_smile:

Reply:
Could we maybe add it too `http://localhost:3000/www.ldbc.eu/tags`? Or should we not create our own directories in that domain?